opendata_module/opmon_opendata/api/helpers.py

```diff
@@ -232,3 +239,32 @@ def get_harvest_rows(
         constraints=constraints, columns=columns, order_by=order_by, limit=limit, offset=offset)
 
     return data_cursor.fetchall(), columns, total_query_count
+
+
+def prepare_data_columns(field_descriptions: Dict[str, Dict[str, str]]) -> List[ConstraintMetaType]:
+    """
+    Prepares data columns based on the field descriptions.
+
+    :param field_descriptions: A dictionary containing field descriptions.
+    :return: A list of ConstraintMetaType objects representing the prepared data columns.
+    """
+    postgres_to_python_type = {'varchar(255)': 'string', 'bigint': 'integer', 'integer': 'integer',
```

Comment:
Have you considered the cases when varchar might be different? Like 254?

Reply:
No, I did'nt, but I see `varchar(255)` used in  Metrics system by default. Not sure if `varchar(254)` or `varchar(256)` should be supported as well.